ai/src/train.py

```diff
@@ -163,18 +219,22 @@ def train(self):
                 end_of_episode(episode)
 
             if self.custom_training_goal_met:
-                print_event(self.pod_name, f"Training goal '{self.training_goal}' reached!")
+                print_event(
+                    self.pod_name, f"Training goal '{self.training_goal}' reached!"
+                )
             elif self.not_learning_episodes_threshold_met:
-                print_event(self.pod_name, "Training goal 'score_variance < 1' reached!")
+                print_event(
+                    self.pod_name, "Training goal 'score_variance < 1' reached!"
+                )
             else:
-                print_event(self.pod_name, f"Max training episodes ({self.training_episodes}) reached!")
-
-        tmpdir = tempfile.mkdtemp(prefix="spiceai_")
-        save_path = Path(tmpdir, f"{self.pod_name}_train")
-        save_path.mkdir(parents=True)
-        directories_to_delete.append(tmpdir)
-        self.agent.save(save_path)
-        self.SAVED_MODELS[self.pod_name] = save_path
+                print_event(
+                    self.pod_name,
+                    f"Max training episodes ({self.training_episodes}) reached!",
+                )
+
+        data_path = Path(self.training_data_dir)
+        self.agent.save(data_path)
```

Comment:
Why getting rid of the temp data? I liked the idea of saving on RAM and not wearing down the SSD with I/O that might just needs to be discarded latter

Reply:
Temp data still exists, it's just managed by the spiced process now instead of the Python one.